packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,232 @@
+		if ( !result.found ) {
+			console.log( 'No files were found.\n' );
+		} else if ( result.found && !result.updated ) {
+			console.log( `${ chalk.bold( result.found ) } files were found, but none needed to be updated.\n` );
+		} else {
+			console.log( `Out of ${ chalk.bold( result.found ) } files found, ${ chalk.bold( result.updated ) } were updated.\n` );
+
+			if ( pathToUpdate.commit ) {
+				totalResult.toCommit += result.updated;
+				pathsToCommit.push( path );
+			}
+		}
+	}
+
+	if ( dryRun ) {
+		console.log( chalk.yellow( 'DRY RUN mode - press any key to display next file diff, or Q to exit.' ) );
+
+		readline.emitKeypressEvents( process.stdin );
+		process.stdin.setRawMode( true );
+
+		process.stdin.on( 'keypress', ( str, key ) => {
+			if ( key.name === 'q' ) {
+				console.log( chalk.yellow( 'Manual exit.' ) );
+				process.exit();
+			}
+
+			const nextDiff = totalResult.differences.shift();
+			const formattedDiff = formatDiff( nextDiff.content );
+
+			console.log( chalk.underline( nextDiff.file ) );
+
+			for ( const line of formattedDiff ) {
+				console.log( line );
+			}
+
+			if ( !totalResult.differences.length ) {
+				console.log( chalk.yellow( 'No more files.' ) );
+				process.exit();
+			}
+
+			console.log( chalk.yellow( 'Q - Exit | Any key - Next' ) );
+		} );
+	} else {
+		if ( pathsToCommit.length ) {
+			console.log( '\n📍 ' + chalk.blue( 'Committing the changes...\n' ) );
+
+			for ( const path of pathsToCommit ) {
+				const execOptions = {
+					stdio: 'inherit',
+					cwd: path
+				};
+
+				console.log( `${ chalk.green( '+' ) } ${ path }` );
+
+				execSync( `git add ${ path }`, execOptions );
+				execSync( 'git commit -m "Internal: Updated all CKEditor 5 dependencies ' +
+				'in `packages/*` to the latest version. [skip ci]"', execOptions );
+			}
+
+			console.log( '\n📍 ' + chalk.green( `Successfully committed ${ totalResult.toCommit } files!\n` ) );
+		}
+
+		if ( totalResult.updated ) {
+			console.log( '\n📍 ' + chalk.green( `Updated total of ${ totalResult.updated } files!\n` ) );
+		} else {
+			console.log( '\n📍 ' + chalk.green( 'No files needed an update.\n' ) );
+		}
+	}
+};
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in the specified directory to the latest version.
+ * Latest version is taken from the `version` property from the root of the `package.json` file, since that value
+ * should already have been bumped at this point of release process.
+ *
+ * @param {String} pathToUpdate Directory containing files to update.
+ * @param {Boolean} dryRun If set to true, diff of changes that would be made is calculated, and included in the returned object. Without
+ * this flag, file is updated normally.
+ * @returns {Object} Object containing numerical values counting files found and updated, as well as array of all changes made. If `dryRun`
+ * is set to true, `differences` array will be empty, as there is no point in calculating that which is only used during `dryRun`.
```

Comment:
It would be nice to re-phrase it a bit to add also the property names that will hold the described values.

Also there is bug in this description: if the `dryRun` flag is `false` (off), but not `true`, as it is now, the `differences` array will be empty.